forte/mrdsrg-helper/dsrg_source.h

```diff
@@ -50,6 +50,29 @@ class DSRG_SOURCE {
     /// Renormalize denominator
     virtual double compute_renormalized_denominator(const double& D) = 0;
 
+    double compute_denominator(const double& D, int i) {
```

Comment:
This is more or less implemented in `STD_SOURCE`. You should probably create a function here, something like
```
/// Derivative of renormalized denominator with respect to Delta
virtual double compute_renormalized_denominator_deriv(const double& D) {
    throw std::runtime_error("Not implemented. Child class should override this function.");
}
```
And then implement in `STD_SOURCE`.